packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md

```diff
@@ -0,0 +1,107 @@
+| `APP_ADMIN`                             | Application Administrator           | Apps                                  |
+| `GROUP_MEMBERSHIP_ADMIN`                | Group Membership Administrator      | [Groups](/docs/reference/api/groups/) |
+| `HELP_DESK_ADMIN`                       | Help Desk Administrator             | [Groups](/docs/reference/api/groups/) |
+| `MOBILE_ADMIN`                          | Mobile Administrator                |                                       |
+| `ORG_ADMIN`                             | Organizational Administrator        |                                       |
+| `READ_ONLY_ADMIN`                       | Read-Only Administrator             |                                       |
+| `REPORT_ADMIN`                          | Report Administrator                |                                       |
+| `SUPER_ADMIN`                           | Super Administrator                 |                                       |
+| `USER_ADMIN`                            | Group Administrator                 | [Groups](/docs/reference/api/groups/) |
+
+Standard role assignment is done in two steps:
+1. Assign a role to a user or group. At this point the admin has the supported privileges fo the role over all resources across organization.
+2. (Optional) If the role supports targets, use one of the [target operations](/docs/reference/api/roles/#role-target-operations) to specify which specific resource the admin can manage.
+
+Note that the entities involved in standard role assignment are:
+* a role; identified either by type or id returned from [listing API](/docs/reference/api/roles/#list-roles)
+* a principal; either a group or a user
+* (Optional) a resource; when using target [target operations](/docs/reference/api/roles/#role-target-operations) this could be either an app or a group
+
+## Custom Role Assignment
+Custom roles can be built by piecing [permissions](/docs/reference/api/roles/#permission-types) together. Once a custom role is built, you can use its id or name to assign to admins. The process is:
+1. Build a custom role
+2. Build a set of resources
+3. Bind the admin with the role from step 1 targeting the resource set from step 2
+
+An assignment of a role to an admin is called a [binding](/docs/reference/api/roles/#binding-object). Conceptually, a binding is identified by the role id. A maximum of one role per resource set will be allowed. Assignment of more admins to the same role targetting the same resource set is possible and they are added as [members]((/docs/reference/api/roles/#member-object)) of the same binding. Hence a binding could simply be seen as:
+```json
+{
+  "${roleId}": [
+    "${memberId1}",
+    "${memberId2}",
+    ...
+  ]
+}
+```
+
+Therefore, when dealing with custom roles, three entities are always in play:
+* a role; identified by its name or id
+* a principal; either a group or a user
```

Comment:
This should be defined further above (probably best in process step #3 "Bind the admin - known as the 'principal' with the role...")

Reply:
(ignore this comment, I see you refer to principal at the very top, and I added a comment there to further explain)